<launch>
    <arg name="init_x"                     value="-5.14"/>
    <arg name="init_y"                     value="-2.17"/>
    <arg name="init_z"                     value="0"/>
    <arg name="init_theta"                 value="-2.8"/>
    <param name="init_x"                   value="$(arg init_x)"/>
    <param name="init_y"                   value="$(arg init_y)"/>
    <param name="init_z"                   value="$(arg init_z)"/>
    <param name="init_theta"               value="$(arg init_theta)"/>
    <node name="init_pose" pkg="four_wheels" type="init_pose"/>
</launch>
